modules/gcp/dc/new_domain_users.ps1.tpl

```diff
@@ -26,13 +24,12 @@ foreach ($User in $ADUsers)
     $Isadmin    = $User.isadmin
 
     #Check to see if the user already exists in AD
-    if (Get-ADUser -F {SamAccountName -eq $Username})
-    {
+    if (Get-ADUser -F {SamAccountName -eq $Username}) {
         #If user does exist, give a warning
-        Write-Warning "A user account with username $Username already exist in Active Directory."
-    }
-    else
-    {
+        Write-Warning "WARNING: A user account with username $Username already exist in Active Directory."

```

Comment:
if we use Write-Warning for warning, is normal output for Error ok or is there a Write-Error?  Will these affect how they show / not show in our logs?